android/src/main/java/com/oney/WebRTCModule/PeerConnectionObserver.java

```diff
@@ -52,10 +55,31 @@
     PeerConnectionObserver(WebRTCModule webRTCModule, int id) {
         this.webRTCModule = webRTCModule;
         this.id = id;
+        this.localStreams = new HashMap<String, MediaStream>();
```

Comment:
After removing the unnecessary methods, we don't need a `HashMap` because all we care about is reference equality. Besides, `PeerConnection` maintains a `List` itself.